src/framework/ui/internal/uiconfiguration.cpp

```diff
@@ -242,33 +250,58 @@ void UiConfiguration::notifyAboutCurrentThemeChanged()
     m_currentThemeChanged.notify();
 }
 
+static QColor colorFromHex(const QString& hex)
+{
+    if (!hex.startsWith('#')) {
+        return QColor();
+    }
+
+    QString str = hex;
+    if (str.length() == 9) { // #RRGGBBAA
+        // Convert to Qt notation: #AARRGGBB
+        QString alpha = str.mid(7, 2);
+        str.chop(2);
+        str.insert(1, alpha);
+    }
+
+    return QColor::fromString(str);
+}
+
 ThemeInfo UiConfiguration::makeStandardTheme(const ThemeCode& codeKey) const
 {
     ThemeInfo theme;
     theme.codeKey = codeKey;
 
-    Config config = ConfigReader::read(QString(":/configs/%1.cfg").arg(QString::fromStdString(codeKey)));
+    io::path_t themeFilePath = globalConfiguration()->appDataPath() + codeKey + ".cfg";
+
+    if (io::FileInfo::exists(themeFilePath)) {
+        m_themeWatcher.startWatching(themeFilePath.toStdString());
```

Comment:
What is the need for a watcher?

Reply:
Maybe this should be added only to the debug build? 
I think tracking file changes is a non-trivial operation :)